vector/src/main/java/im/vector/app/features/home/room/threads/list/views/ThreadListFragment.kt

```diff
@@ -104,6 +104,11 @@ class ThreadListFragment @Inject constructor(
     override fun invalidate() = withState(threadListViewModel) { state ->
         renderEmptyStateIfNeeded(state)
         threadListController.update(state)
+        renderLoaderIfNeeded(state)
+    }
+
+    private fun renderLoaderIfNeeded(state: ThreadListViewState) {
```

Comment:
`IfNeeded` suffix is not necessary here I guess.

Reply:
Correct, its there from a previous approach!